src/go/cmd/module.go

```diff
@@ -54,6 +54,20 @@ func NewModule(dir string) (*Module, error) {
 		}
 		fmt.Printf("Package Name: %s\n", packageName)
 	}
+
+	//Package name can still be empty when generating API review using uploaded zip folder of a specific package in which case parent directory will not be sdk
+	if packageName == "" {
+		modulePath := mf.Module.Mod.Path
+		packageName := path.Base(modulePath)
+		fmt.Printf("Module path: %s\n", modulePath)
+		// Set relative path as package name for internal package to avoid collision
+		if packageName == "internal" {
```

Comment:
This is special casing for modules like `sdk/internal`, `sdk/keyvault/internal` etc yes?

Reply:
correct. Otherwise, all internal packages will get same "internal" as name.